index.html

```diff
@@ -20,7 +20,7 @@ <h2 class="greetings">Glad to meet you!</h2>
                 <p>If you will like what you see and have a project you will need coded,don't hesitate to contact me</p> 
                 <p>look through some of my work and experince</p>
             </div>
-            <h3 class="heading" style="color:blue;">LET'S CONNECT</h3>
+            <h3 class="heading">LET'S CONNECT</h3>
             <div class="icon-bar">
                 <img src="/home/ignatius/Documents/Microverse Projects/Portfolio/images/twitter.png" alt="">
                 <img src="/home/ignatius/Documents/Microverse Projects/Portfolio/images/linkedin.png" alt="">
```

Comment:
I suggest you fix the paths to this images since they are not showing. This is your design
<img width="703" alt="Screenshot 2023-01-14 at 08 40 14" src="https://user-images.githubusercontent.com/48546992/212458444-cdb1a5d0-7c7b-4971-8596-aa0ff2bad2a2.png">
this is the figma template
<img width="373" alt="Screenshot 2023-01-14 at 08 39 58" src="https://user-images.githubusercontent.com/48546992/212458456-5b678db6-7bb6-4f7c-a767-23394db603f3.png">

Also i suggest instead of using plain html img elements, you make this images links using the `<li> <a><img>` syntax. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.